plugins/mouse/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ set(PANEL_SOURCES
     ${QML_SOURCES}
 )
 
+configure_file(qmldir.in qmldir)
```

Comment:
Since these does not "configure" anything, please set COPYONLY https://cmake.org/cmake/help/latest/command/configure_file.html

Reply:
Done.